<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>595</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>정선동강</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Motion Palette</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1001</width>
     <height>571</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Btn_com_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>530</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>771</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="L_layout"/>
   </widget>
   <widget class="QPushButton" name="Btn_del_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>280</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>전체 삭제</string>
    </property>
   </widget>
   <widget class="QListView" name="used_list">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="Brush_On">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>79</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>붓 활성화</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>선 색상</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>선 굵기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>36</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>리스트</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_color">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="Line_tk">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Btn_paint_type2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>도안위에그리기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_paint_type1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>명화위에그리기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_com_1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>530</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>저장하기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 이름</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>320</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>휘날리는</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="Combo_effect1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>눈</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>돈</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>물감</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>물방울</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>벚꽃</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>잎</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하트</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Btn_effect1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>넣기</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>360</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Btn_effect2_motion_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>모션 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect2_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>내리는</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect2_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect2_color">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect2_motion_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>모션 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>430</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Btn_effect3_motion_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>모션 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect3_color">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect3_motion_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>모션 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect3_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect3_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>퍼져가는</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>500</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Btn_effect4_motion_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>모션 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect4_color">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect4_motion_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>모션 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect4_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_effect4_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>떠다니는</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Btn_del_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>280</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Effect2_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[ 내리는 ] :  00 </string>
     </property>
    </widget>
    <widget class="QLabel" name="Effect3_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[ 퍼져가는 ] :  00</string>
     </property>
    </widget>
    <widget class="QLabel" name="Effect4_label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[ 떠다니는 ] :  00</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_6</zorder>
   <zorder>frame_2</zorder>
   <zorder>Btn_com_2</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>Btn_del_2</zorder>
   <zorder>used_list</zorder>
   <zorder>Brush_On</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>Btn_color</zorder>
   <zorder>Line_tk</zorder>
   <zorder>Btn_paint_type2</zorder>
   <zorder>Btn_paint_type1</zorder>
   <zorder>Btn_com_1</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>fileName</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame_4</zorder>
   <zorder>frame_5</zorder>
   <zorder>Btn_del_1</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
